core/test/results/sample_v2.json

```diff
@@ -4089,13 +4089,6 @@
         "items": []
       }
     },
-    "target-size": {
```

Comment:
Isn't audit removal, even if it's a hidden audit a breaking change?

Reply:
I agree we need top stop running it to temporarily fix the bug, but we don't need to remove from the config so your second suggestion SGTM.